.gitignore

```diff
@@ -14,3 +14,6 @@ coverage.txt
 
 # Dependency directories (remove the comment below to include it)
 # vendor/
+
+# Direnv 
+.envrc
```

Comment:
Add this to your global git ignore file and undo this change please.

Reply:
👍 Done